src/student-store-gcs.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "student-store-359018",
+  "private_key_id": "79c5d85e958a06949ca053e57cc02c77c3fc072c",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQDAuQcS/za7/XEO\nNEIL6DXWBdB46uU/GTMY+2cEL/lSnF/oBuFh3yIpLRt58rL5zqK40G/z5Zf9MkVB\nhe/+o3nYmyfRyKy+8TvvMMvz34OUkKkhMawoE9TvDjVRvKORoaoJuI8PkMD/bcgG\nVwNHwn2m8NSD2B9faV3UFAH6hC1ot6j7T4Ms5p8KRol6XtDOuoum2zy/RvABnqpn\nORZdMtGgFt2zI1dQbK4sT56TNJEWoNgCowdRiziqzFJBvc9hgJQhYg1EoDXeMeyp\n8tXyc5r+G7Rad03pPzALxgIRv45L7lxzF7YHx68kXoLbuQIiykPLXpaU6EoeaWxB\n/pS15bjvAgMBAAECggEAPddBh12AjTsqm3G2XcbB+6668goBzMkDxfjp2gSg3trR\nxYzrpnGorwjL38o6lwEw4tfzjJI4PqVyZq+ZTp7bWt+Sm/TmRZD39+0zDMrEmJgA\nxWeAhA21Do6DGnrLC9CR1PO1aDu9SllKTYp/QXgNxNNThgkELCQN2R8gcWgXgBmb\nbkwdNF5ZcN5yhVh/AmXuZ9wQIR1ELrQA5+UMCVHgq/rj18Z2MUswO8r5oy/pqrV3\nuznx3sydOkXjWNVi/es4aOShIe4R2oG70fkUHjOJXkyg4vofQK/CKyVwGYro5W0X\nCfNVghcuaAM7dJ93YN5Vx8ky5p4itFDd32FlxeRz+QKBgQD5SRjOKxFtcMO5k4SF\nNZ15f9SQYPSNxL1EkQRUW/pCSPhNIt1qeSj6QBxcUfJXEBZzDG+WW4fsUI4lhNMq\no44y5nTY/y3LsW82+675fmXWfVtA0Sa3/YFTnrmpcq2DUtsmIWdisGaE/rwmKJ7c\np+gs0YWtEtemAVIQFaMqc/rueQKBgQDF6emVJk9FSyPhH59/6hqPOvdodQU0/u8Y\n/MpIxAIXGwl1TiyRg1rWhpSXhXcroph2IUGK/aPC4tOk4u/w6cSziHI3XNLM0iQT\nD2ddGq/o9tVjIzdx3yqZ+o/639to1Oh6XJdxlLZ5xttSXC9oPaYDJ/8wZVdrU0+D\nd0Zb/itopwKBgH2cXppsSzDJNGaLNahl1r7FMde6cyAIdZJaPENBwX2rBSrhGY6w\nFHzc1y2RwTorihRYvNWXs4rZIHc5UapXlayCQXiiRHIyKycadRWUoj4NFnE3snnd\njFAVV9pRRas5keHJzS2GU3JKZYYYy8vF85C51tDaAL3ORoqCTKB0pomJAoGBAJ/y\npEFS/butt2iIaxows5hP+oVkFXo4Ejnqivz11A6iXil8rU5Ij1BS4aH3TlFojbji\ntZrD0+Z79LoMYggclUjXHB+cGA1KQ7dXupynJcWsPtZpLrULu91Hf7cr1aHKd7v0\nOryjiDmwua29LN8c+58PYxDUxirJ4LgJ3EYaizB/AoGBAKHdTHKbhfJaZD2j6I5e\nUC6TRHKo0bQ//AE538cYEAQQYAVHhMZabKHXmiIW2/xgBe4eVpjWn/hZQ1Ma9miN\nspXVu3IURF9s1VXs8pHIJzDdY+orRIcf6kZq2SxqZU+NahyekooB7qapoDidMDDh\nMgm3OCN5vYGOQAOkdaN5AYZz\n-----END PRIVATE KEY-----\n",
+  "client_email": "student-store-service-account@student-store-359018.iam.gserviceaccount.com",
+  "client_id": "116769893538074891742",
+  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+  "token_uri": "https://oauth2.googleapis.com/token",
+  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/student-store-service-account%40student-store-359018.iam.gserviceaccount.com"
+}
```

Comment:
Such credentials shouldn't be stored at the repo.

Reply:
yeah, accidentally uploaded this file. Forgot to add it to .gitignore.